src/me/goddragon/teaseai/api/picture/TagsFile.java

```diff
@@ -168,20 +170,21 @@ public boolean setDressState(DressState dressState, File image) {
                     inputBuffer.append(strLine);
                     inputBuffer.append('\n');
                 }
-
-                lines.set(lines.indexOf(replaceLine), newLine);
+                TeaseLogger.getLogger().log(Level.INFO, "Added = " + added);
+                if (added) {
```

Comment:
You can change this into an else statement with the already existing !added statement

Reply:
yea. Thats fine and remove that logger message. That was just for testing.